### PR TITLE
Add ignore option with [map]->Record default

### LINKED ISSUE #92: Marking a record as ^:no-doc still shows the constructor functions (->Foo and map->Foo)
It seems to me that if you are hiding the existence of the record, your should hide the constructors as well.  Make sense?

Observed in 0.8.12.

### PR DESCRIPTION
**Pull request by floybix**
Fixes #92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,14 @@ To override the namespaces list and include all namespaces, use `:all`
 :codox {:namespaces :all}
 ```
 
+The `:ignore` option can be used to filter out vars whose name matches a
```
Comment (weavejester) [inline README.md]: Let's call it `:exclude-vars` rather than `:ignore`.

File: codox/src/codox/writer/html.clj [hunk 2]
```diff
@@ -467,10 +467,20 @@
 (defn- apply-theme-transforms [{:keys [themes] :as project}]
   (reduce apply-one-theme project themes))
 
+(defn- apply-ignore [{:keys [ignore] :as project}]
```
Comment (weavejester) [inline codox/src/codox/writer/html.clj]: This should be in the reader, rather than the writer.

**Comment by floybix** [comment]
Ok, updated

**Comment by floybix** [comment]
Fixes #77 also (duplicate)

**Pull request merged**